back/spec/acceptance/project_custom_fields_spec.rb

```diff
@@ -49,36 +49,32 @@
       example_request 'Get the react-jsonschema-form json schema and ui schema for the custom fields' do
         expect(status).to eq 200
         json_response = json_parse(response_body)
-        expect(json_response[:json_schema_multiloc].keys).to eq %i[en fr-FR nl-NL]
+        expect(json_response[:json_schema_multiloc].keys).to match_array %i[en fr-FR nl-NL]
         %i[en fr-FR nl-NL].each do |locale|
-          expect(json_response[:json_schema_multiloc][locale][:properties].keys).to eq(
-            %i[title_multiloc body_multiloc author_id budget topic_ids location_description idea_images_attributes idea_files_attributes]
```

Comment:
For the new schema, this is tested in the json_forms_service_spec. The author and budget fields are not present when they should not be included in the form.

However, I didn't think about the old schema. This schema was disabled everywhere and so far, we didn't get any complaints about issues with the new schema. Perhaps we could move forward by removing old schema support for input forms?

I think it would probably be good to move the registration form to the new schema (I don't know if there is a reason on the frontend side to still use the old one?) as well. I think the flexible registration epic could be a good opportunity to do this.